README.md

```diff
@@ -1,15 +1,18 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
 # Project Name
+TO-DO-LIST
 
 > Description the project.
+This project is a simple To-do-List-App which user adds and removes tasks using HTML/CSS to build the app and JavaScript to add functionalities to the app by using webpack dev server to display the app on a web server.
 
 
 ## Built With
```

Comment:
- [ ] Please add a screenshot of your app so as to give visitors a quick preview of what your app looks like 😃 👍